src/main/java/com/kryptnostic/datastore/services/EdmService.java

```diff
@@ -380,6 +381,7 @@ public void renamePropertyType( UUID propertyTypeId, FullQualifiedName newFqn )
     public void renameEntitySet( UUID entitySetId, String newName ) {
         aclKeyReservations.renameReservation( entitySetId, newName );
         entitySets.executeOnKey( entitySetId, new RenameEntitySetProcessor( newName ) );
+        eventBus.post( new EntitySetMetadataUpdatedEvent( entitySetManager.getEntitySet( newName ) ) );
```

Comment:
would be better to do 
```eventBus.post( new EntitySetMetadataUpdatedEvent( getEntitySet( entitySetId ) ) );```
instead - the other call is a secondary index lookup which could be expensive.